src/main/java/tds/exam/results/services/impl/SessionServiceImpl.java

```diff
@@ -27,7 +27,6 @@ public Session findSessionById(final UUID sessionId) {
     }
 
     @Override
-    @Cacheable(CacheType.LONG_TERM)
```

Comment:
Let's leave this long term for the reason explained [here](https://github.com/SmarterApp/TDS_ExamService/pull/201/files#diff-3d1a0d3a15b472a041737be6ab709628L64)